### DISCUSSION TITLE
TED Viewer gives an error

### DISCUSSION THREAD

**Discussion by AnalystEstonia** [OP-TED/eForms-SDK#638] [Q&A] [answered]
Hello! We have a question about the TED Viewer.

The publication service (https://cvs.preview.ted.europa.eu/v1/notices/validation) shows that there are no validation errors regarding the XML, but at the same time the generation of the notice file (HTML/PDF), render service {}({}https://viewer.preview.ted.europa.eu/v1/render{}){_} gives an error.

[example.txt](https://github.com/OP-TED/eForms-SDK/files/12321209/example.txt)

**Comment by pcolast** [comment, marked as answer]
Good day,

Thank you for spotting it and providing an example which causes rendering to fail.
The issue has been identified.

We will inform you when the fix has been deployed.

We apologize for the inconvenience.

Have a nice day,
Stefan

**Comment by pcolast** [comment]
Dear users,

A new release of TED Viewer was installed on the 4th October in Preview which goes into Production on the 5th October. This should solve the issue you were encountering.

Have a nice day,
Stefan
